File: C#/Iniciante/1014 - Consumo.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 12:43:53
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int x;
            double y;
            x = int.Parse(Console.ReadLine());
            y = double.Parse(Console.ReadLine());
            Console.WriteLine($"{(x/y).ToString("F3", CultureInfo.InvariantCulture)} km/l");
        }
    }
}

File: Java/Iniciante/1014 - Consumo.java
/*
* Autor: Pierre Vieira
* Data de submissão: 04/03/2020 12:40:28
*/
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        int x;
        float y;
        x = teclado.nextInt();
        y = teclado.nextFloat();
        System.out.printf("%.3f km/l\n", x/y);
    }

}
